### PR TITLE
[fix #3] Adding AppIdentifier header in example

### LINKED ISSUE #3: Missing AppIdentifier header
PHP Examples https://github.com/e-conomic/eco-api-ex/blob/master/examples/php/list_debtors.php#L81 are missing `AppIdentifier header`
```
IMPORTANT - Don't lose your connection!
AppIdentifier header is soon required on SOAP for Connect(), ConnectAsAdministrator() and ConnectAsAdministratorWithCustomerNumber()
From March 15th 2016 we require that all SOAP API requests include a valid X-EconomicAppIdentifier header.
Read more here http://techtalk.e-conomic.com/e-conomic-soap-api-now-requires-you-to-specify-a-custom-x-economicappidentifier-header/
```

### PR DESCRIPTION
**Pull request by gunjanpatel**
Adding AppIdentifier header.
This will fix issue #3

### CODE REVIEW

File: examples/php/list_debtors.php [hunk 1]
```diff
@@ -1,233 +1,474 @@
...
+		array(
+			"trace" => 1,
+			"exceptions" => 1,
+			"stream_context" => stream_context_create(
+				array(
+					"http" => array(
+						"header" => "X-EconomicAppIdentifier: MyCoolIntegration/1.1 (http://example.com/MyCoolIntegration/; MyCoolIntegration@example.com) BasedOnSuperLib/1.4"
+					)
+				)
+			)
```
Comment (gunjanpatel) [inline examples/php/list_debtors.php]: Added above lines for fix.

File: examples/php/list_debtors.php [hunk 2]
```diff
@@ -1,233 +1,255 @@
...
+		}
+		catch(Exception $exception)
+		{
+			print("<p><b>Error fetching orders for the selected debtor.</b></p>");
+			print("<p><i>" . $exception->getMessage() . "</i></p>");
+		}
+	}
+?>
+<h1>Create debtor</h1>
+<form action="<?php echo $me . "?token=" . $_REQUEST['token'] . "&appToken=" . $_REQUEST['appToken'];?>" method="post">
```
Comment (olejessien) [inline examples/php/list_debtors.php]: This is the credentials based connect example. Why would we reference tokens?

File: examples/php/list_debtors.php [hunk 3]
```diff
@@ -1,233 +1,255 @@
...
+			<td><b>Name</b></td>
+			<td><b>Address</b></td>
+			<td><b>PostalCode</b></td>
+			<td><b>City</b></td>
+			<td><b>Country</b></td>
+			<td class="white_field"></td>
+		</tr>
+	<?php foreach ($debtorDataObjects as $i => $debtorData) : ?>
+		<tr class="<?php if($i % 2 == 0) echo 'even_row'; else echo 'odd_row' ?>">
+			<form action="<?php echo $me . "?token=" . $_REQUEST['token'] . "&appToken=" . $_REQUEST['appToken'] . "&password=" ;?>" method="post">
```
Comment (olejessien) [inline examples/php/list_debtors.php]: This is the credentials based connect example. Why would we reference tokens?

**Comment by olejessien** [comment]
Hi Gunjan
Thank you for contributing. That's really awesome :-)
Could you please review my comments regarding what looks like a mixup between the token/non-token examples? I'll give it a spin afterwards.

File: examples/php/list_debtors.php (hunk 3 above)
Comment (gunjanpatel) [inline examples/php/list_debtors.php]: ahh my bad. Thanks.

**Comment by gunjanpatel** [comment]
Welcome @olejessien I have fixed your comments.

File: examples/php/list_debtors_with_token.php [hunk 4]
```diff
@@ -27,427 +21,237 @@
...
+			"exceptions" => 1,
+			"stream_context" => stream_context_create(
+				array(
+					"http" => array(
+						"header" => "X-EconomicAppIdentifier: MyCoolIntegration/1.1 (http://example.com/MyCoolIntegration/; MyCoolIntegration@example.com) BasedOnSuperLib/1.4"
+					)
+				)
+			)
+		)
+	);
```
Comment (olejessien) [inline examples/php/list_debtors_with_token.php]: AppIdentifier is only needed on credentials based connects. Token connect (ConnectWithToken) does not require AppIdentifier as the token App and the developer agreement serve this purpose.

Comment (gunjanpatel) [inline examples/php/list_debtors_with_token.php]: Good to know. Thanks.

Comment (olejessien) [inline examples/php/list_debtors_with_token.php]: I'll update the token example later to reflect this. :-)

**Pull request merged**